From d7682af8ba3f3675dbede8e19778d9b466e4363b Mon Sep 17 00:00:00 2001
From: agaraix <anath.bandux.garai@intel.com>
Date: Sat, 18 Jan 2020 21:40:18 +0530
Subject: [PATCH] UGW_SW-42634 : PPA session mtu fix

Separate APIs used for a net_device mtu and base mtu.
---
 include/net/ppa/ppa_stack_al.h | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/include/net/ppa/ppa_stack_al.h b/include/net/ppa/ppa_stack_al.h
index 426fd989f0e1..3c15bf853ee5 100644
--- a/include/net/ppa/ppa_stack_al.h
+++ b/include/net/ppa/ppa_stack_al.h
@@ -1380,6 +1380,12 @@ PPA_NETIF *ppa_get_netif (const PPA_IFNAME *ifname);
 PPA_NETIF *ppa_get_netif_by_net(const struct net *net, const PPA_IFNAME *ifname);
 
 /*#ifdef CONFIG_LTQ_MINI_JUMBO_FRAME_SUPPORT*/
+/*! \brief This function returns the base MTU of the interface.
+	\param[in] netif Pointer to the netif structure.
+	\return MTU size.
+	\note
+ */
+int ppa_get_base_mtu(PPA_NETIF *netif);
 /*! \brief This function returns the MTU of the interface.
 	\param[in] netif Pointer to the netif structure.
 	\return MTU size.
